examples/jsm/postprocessing/OutlinePass.js

```diff
@@ -4,7 +4,6 @@ import {
 	DoubleSide,
 	LinearFilter,
 	Matrix4,
-	MeshBasicMaterial,
```

Comment:
Since `this.maskBufferMaterial` is removed, we don't need to import MeshBasicMaterial. So I remove it as well.